eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -0,0 +1,39 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# This is to help with testing the release pipeline.
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
+$latestTags = git tag -l "azure-template*"
```

Comment:
I'd suggest `azure-template_*` just in case we have other packages that end up starting with azure-template like we do in java.